utilities/GitUtilities.groovy

```diff
@@ -75,13 +75,29 @@ def getCurrentGitDetachedBranch(String gitDir) {
 	String gitBranchString = gitBranch.toString()
 	def gitBranchArr = gitBranchString.split(',')
```

Comment:
For readability purpose, can you rename the `gitBranchArr` variable to something like `gitBranches`? Defining it as `String[]` would also help, but it's not required in groovy...